_posts/2019-07-15-hccamp.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title: "HolidayCheckCamp 2019"
+date: 2019-07-15 12:0:00 +0200
+categories: event
+author_name: Álvaro Nistal Valbuena
+author_url : author/alvaronistalvalbuena/
+author_avatar: alvaronistalvalbuena
+read_time : 5
+excerpt: 
+feature_image: posts/2019-07-15-hccamp/header.jpg
+
+---
+
+# HolidayCheckCamp 2019
+
+As part of our mission of having the best team, last May we celebrated in the Golden Sands of Bulgaria the first edition of the HolidayCheck Camp. Around 100 people of the Product, IT, UX & Design and Agile team from 27 nationalities and 4 different office locations (Bottighofen, Munich, Poznań and Amsterdam) enjoyed a two-days conference. 
+
+This is the only time of the year where the whole Product/IT Organisation gets the chance to meet in one place together, sit together, work together and look for ways to improve HolidayCheck's products. We review the way we work as a team and have a look back at all the good and bad things we have done during the last year. Since most of our teams are cross-location, this is a perfect opportunity to grow as one team.
+
+## How it all began
+
+With the goal of building one strong IT, two years ago our whole IT met during a couple of days to discuss and share views on different topics around HolidayCheck, IT, design, the travel industry and much more. This get-together was knwon as AllDevs and was the beginning of a great tradition.
+
+This has evolved during last two editions involving more departments related to the IT organisation (Product, UX, Agile), different locations, colleagues from other companies (this last edition friends joined us from DriveBoo, Weeronline and Workpath) and new conference styles. And that brings us to what we are today, the HolidayCheckCamp or HC.Camp.
+
+But why calling it a Camp?
+
+- the camp is a place to meet, share and discuss.
+- it introduces the nomadic view of the conference; each year, one different location.
+- the naming does not limit the participants to being only devs.
+
+## HolidayCheckCamp 2019
+
+This last edition has taken place in Bulgaria, at the Golden Sands, in the HolidayCheck Award 2019 [Hotel Griffid Encanto Beach](https://www.holidaycheck.de/hi/grifid-encanto-beach/22201c5d-e735-3966-803a-b44572669ffa). Due to the previous work of an amazing Orga Team, the HC.Camp was an awesome experience for me, both as organiser and participant. 
+
+(missing some text here)
+
+The conference itself consisted of 2 different tracks: Campfire and Compass.
```

Comment:
`The conference itself consisted of 2 different tracks: Campfire and Compass.` ->
`The conference itself consisted of 2 tracks, Campfire and Compass. How were they different?`